src/Command/CreateCommand.php

```diff
@@ -30,11 +30,11 @@
  *
  * This command creates a new migration using the available migration template.
  *
- * Config in di-container console.php migrations paths `createPath` and `updatePaths`:
+ * To use it, configure migrations paths (`createPath` and `updatePaths`) in DI container via `console.php`:
```

Comment:
That looks like too much for command phpdoc. Could it be moved to docs?

Reply:
fixed.